src/ui/public/doc_table/components/table_row.js

```diff
@@ -128,7 +147,7 @@ module.directive('kbnTableRow', function ($compile) {
           // rebuild cells since we modified the children
           $cells = $el.children();
 
-          if (i === 0 && !reuse) {
+          if (!reuse) {
             $toggleScope = $scope.$new();
             $compile($target)($toggleScope);
```

Comment:
@marenas I haven't personally seen any slowdown with this change. The template for each cell is pretty minimal.